detectable/src/test/java/com/synopsys/integration/detectable/detectables/go/vendr/GoVndrDetectableTest.java

```diff
@@ -20,7 +20,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package com.synopsys.integration.detectable.detectables.go.functional;
+package com.synopsys.integration.detectable.detectables.go.vendr;
```

Comment:
I moved this Detectable test from the `functional` package to the `vendr` package.
Personally I don't derive any value from splitting `functional` and `unit` tests into their own packages.
Additionally it is a pain point since creating a test in IntelliJ will automatically put it in the same package as the class.
Also when running coverage reports on this class in IntelliJ, it failed to produce coverage because of the package discrepancy. 
Is there a compelling reason for keeping them separate?